backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/LuceneNumericFieldSortBuilder.java

```diff
@@ -32,13 +34,25 @@
 
 	@Override
 	public void buildAndContribute(LuceneSearchSortCollector collector) {
+		collector.collectSortField( getSortField(), nestedFieldContribution );
+	}
+
+	private SortField getSortField() {
```

Comment:
Looks like your renamed this to `createSortField` in the text field sort builder, but not here.